core-web/libs/dotcms-scss/angular/dotcms-theme/components/_custom-icons.scss

```diff
@@ -0,0 +1,7 @@
+.dot-trophy {
+    height: 1.5rem;
+    width: 1.5rem;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url("data:image/svg+xml;charset=UTF-8,%3csvg width='24' height='24' viewBox='0 0 24 24' fill='none' xmlns='http://www.w3.org/2000/svg'%3e%3cpath fill-rule='evenodd' clip-rule='evenodd' d='M7.62852 1.07129H16.3713C17.1761 1.07129 17.8285 1.72367 17.8285 2.52843V9.81411C17.8285 13.0331 15.2189 15.6426 11.9999 15.6426C8.78091 15.6426 6.17139 13.0331 6.17139 9.81411V2.52843C6.17139 1.72367 6.82377 1.07129 7.62852 1.07129Z' stroke='%23426BF0' stroke-width='1.6' stroke-linecap='round' stroke-linejoin='round'/%3e%3cpath d='M11.9995 15.6426V20.014' stroke='%23426BF0' stroke-width='1.6' stroke-linecap='round' stroke-linejoin='round'/%3e%3cpath fill-rule='evenodd' clip-rule='evenodd' d='M7.62835 20.0148H16.3712C17.1759 20.0148 17.8283 20.6672 17.8283 21.472C17.8283 22.2767 17.1759 22.9291 16.3712 22.9291H7.62835C6.82359 22.9291 6.17121 22.2767 6.17121 21.472C6.17121 20.6672 6.82359 20.0148 7.62835 20.0148ZM17.8283 3.98633H20.7426C21.5473 3.98633 22.1997 4.63871 22.1997 5.44346V6.9006C22.1997 8.51011 20.8949 9.81487 19.2854 9.81487H17.8283V3.98633ZM6.17121 3.98633H3.25694C2.45219 3.98633 1.7998 4.63871 1.7998 5.44346V6.9006C1.7998 8.51011 3.10457 9.81487 4.71408 9.81487H6.17121V3.98633Z' stroke='%23426BF0' stroke-width='1.6' stroke-linecap='round' stroke-linejoin='round'/%3e%3c/svg%3e");
+}
```

Comment:
Should we use the svg file inside an image tag just directly? Angular have the `/assets` folder.